source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -0,0 +1,117 @@
+# Mattermost Cloud Changelog
+
+This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
+
+## Release 2020-12-09
+
+### Highlights
+
+#### Incident Management provided out-of-the-box (E20)
```

Comment:
Do not need (E20) Also wondering if we need this one at all since Incident management has been included since launch.